api/src/main/java/com/codeforcommunity/rest/FailureHandler.java

```diff
@@ -1,24 +1,6 @@
 package com.codeforcommunity.rest;
 
-import com.codeforcommunity.exceptions.CreateUserException;
-import com.codeforcommunity.exceptions.EmailAlreadyInUseException;
-import com.codeforcommunity.exceptions.ExpiredSecretKeyException;
-import com.codeforcommunity.exceptions.HandledException;
-import com.codeforcommunity.exceptions.IncorrectBlockStatusException;
-import com.codeforcommunity.exceptions.InvalidSecretKeyException;
-import com.codeforcommunity.exceptions.LeaderCannotLeaveTeamException;
-import com.codeforcommunity.exceptions.MalformedParameterException;
-import com.codeforcommunity.exceptions.MemberApplicationException;
-import com.codeforcommunity.exceptions.MemberStatusException;
-import com.codeforcommunity.exceptions.MissingHeaderException;
-import com.codeforcommunity.exceptions.MissingParameterException;
-import com.codeforcommunity.exceptions.ResourceDoesNotExistException;
-import com.codeforcommunity.exceptions.TokenInvalidException;
-import com.codeforcommunity.exceptions.UsedSecretKeyException;
-import com.codeforcommunity.exceptions.UserDoesNotExistException;
-import com.codeforcommunity.exceptions.UserNotOnTeamException;
-import com.codeforcommunity.exceptions.UsernameAlreadyInUseException;
-import com.codeforcommunity.exceptions.WrongTeamRoleException;
+import com.codeforcommunity.exceptions.*;
```

Comment:
Aaahh no star

Reply:
I turned this off don't know how it snuck in!